Vagrantfile

```diff
@@ -112,14 +112,34 @@ done
     }
   end
 
+  config.trigger.after :destroy do |trigger|
```

Comment:
```suggestion
  config.trigger.after :destroy do |trigger|
    cmds = [config_macos_fqdn, config_jenkins_fqdn, config_ubuntu_fqdn, config_windows_fqdn].map do |filename|
      return 'rm  -f tmp/#{filename}.ssh_known_hosts'
    end
    trigger.run = {inline: "sh -c '#{cmds.join(' && ')}'"}
  end
```

Reply:
then you can remove the subsequent lines

Reply:
this doesn't run it only for macos when macos is destroyed